providers/src/airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -396,7 +396,9 @@ def __init__(
         self.remote_pod: k8s.V1Pod | None = None
         self.log_pod_spec_on_failure = log_pod_spec_on_failure
         self.on_finish_action = OnFinishAction(on_finish_action)
-        self.is_delete_operator_pod = self.on_finish_action == OnFinishAction.DELETE_POD
+        # The `is_delete_operator_pod` parameter should have been removed in provider version 10.0.0.
```

Comment:
`10.0.0` was released, should we change this version to `11.0.0`?

Reply:
Unfortunately not. Long story short, this parameter wasn't removed properly during the 10.0.0 release.
In more detail:
This parameter removal was announced as a breaking change in the 10.0.0 [release notes](https://airflow.apache.org/docs/apache-airflow-providers-cncf-kubernetes/stable/changelog.html) and it's usage was actually removed everywhere. BUT, the parameter itself remains in the __init__() method.  If I removed it now, then the community would have to bump the cncf-kubernetes provider version once again.
Thus I just left this comment here to attract the community's attention and make them decide what to do with this small mess.